ansible/roles/stack-sunbird/templates/sunbird_discussions-mw-service.env

```diff
@@ -1 +1,4 @@
-discussion_forum_token= {{ discussion_forum_token }}
\ No newline at end of file
+sunbird_discussion_forum={{ discussionsmw_sunbird_discussion_forum }}
+authorization_token={{ discussionsmw_nodebb_authorization_token }}
+mongodb_connection_url={{ discussionsmw_mongodb_connection_url }} # the connection url should be 'mongodb://host:port'
```

Comment:
does it support username and pod

Reply:
@VenkatKadiveti ?

Reply:
yes It supports 
mongodb_connection_url
it supports username and password also